src/pages/[lang]/index.tsx

```diff
@@ -1,18 +1,33 @@
 import type { GetStaticPaths, NextPage } from "next";
 
 import { Infoblock, Hero } from "@pagopa/mui-italia";
+import { Typography } from "@mui/material";
+import { IMAGES_PATH, langCodes, PN_PF_URL, PN_PG_URL } from "@utils/constants";
 
-import {
-  getHeroData,
-  getInfoblockData,
-} from "../../api";
-import { LangCode, UserType } from "../../model";
+import { LangCode } from "../../model";
 import PageHead from "../../components/PageHead";
 import { getI18n } from "../../api/i18n";
-import { langCodes } from "@utils/constants";
 import { useTranslation } from "src/hook/useTranslation";
 
-const USER_TYPE = UserType.PH;
+const onReadClickEnti = () => {
+  window.open("/pubbliche-amministrazioni", "_self");
```

Comment:
for internal links use the router navigation as done in the navigation bar

```
push(`/${lang}${path}`);
```

do this for all the internal links